c_glib/test/test-memory-mapped-file.rb

```diff
@@ -22,9 +22,8 @@ def test_open
     tempfile.close
     file = Arrow::MemoryMappedFile.open(tempfile.path, :read)
     begin
-      buffer = " " * 5
-      file.read(buffer)
-      assert_equal("Hello", buffer)
+      buffer = file.read(5)
+      assert_equal("Hello", buffer.data.to_s)
```

Comment:
That's interesting that you can mutate strings like that in Ruby -- I guess you technically can in Python, too